portafly/doc/ux_specs/product/overview.md

```diff
@@ -0,0 +1,17 @@
+# Product overview
+
+[Back to index](../../index.md)
+
+* [Mockups](https://marvelapp.com/prototype/ee82j74/screen/70939434)
+* [Mockups JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4389)
+* [Parent JIRA ticket](https://issues.redhat.com/browse/THREESCALE-4197)
+
+##### Page header
+* Edit button
+  * Secondary button
+* Delete button
+  * Customized tertiary button
```

Comment:
What's the customization about? Only color? If so, could you add its code/name?

Reply:
@cntlsn Should we use the warning color for this? and the secondary button style?

Reply:
@josemigallas the customization is eventually a temporary thing. a **destructive secondary button** is in the process of being implemented by the patternfly team (https://github.com/patternfly/patternfly/issues/1094#issuecomment-470592366). in the meantime we could just use a primary destructive button, or if easy enough customize a secondary button to resemble a destructive button, aka creating a custom class such as:
```
.pf-c-button.pf-m-secondary-danger {
    --pf-c-button--after--BorderColor: var(--pf-global--danger-color--100);
    color: var(--pf-global--danger-color--100);
    background-color: var(--pf-c-button--m-secondary--BackgroundColor);
}
```